DeepSpeech.py

```diff
@@ -307,8 +307,26 @@ def get_tower_results(iterator, optimizer, dropout_rates):
                     tower_avg_losses.append(avg_loss)
 
                     # Compute gradients for model parameters using tower's mini-batch
-                    gradients = optimizer.compute_gradients(avg_loss)
-
+                    if FLAGS.load == "transfer":
+                        if not FLAGS.fine_tune:
```

Comment:
nit: Improve readability by putting the positive case into the "true" branch of the "if" and the negative case (without "not") into the "else" one.